todo_app/data/todo_items.py

```diff
@@ -0,0 +1,31 @@
+import collections
+from turtle import title
+from typing import Collection
+from unittest import result
+import requests
+import json
+import os
+import pymongo
+from dotenv import load_dotenv
+load_dotenv()
+
+mongo_connection=(os.getenv('MONGO_CONNECTION_STRING'))
+mongo_db=(os.getenv('MONGO_DB_NAME'))
+
+client = pymongo.MongoClient(mongo_connection)
+database = client['todoapp-DB']
```

Comment:
The above said, sometimes we might want to control the order in which things happen but because these sit at the top level of the file, they run automatically as soon as the file is imported. A slightly nicer pattern here might be to capture these in a function and use them like below:

```suggestion
def get_database():
  mongo_connection=(os.getenv('MONGO_CONNECTION_STRING'))
  mongo_db=(os.getenv('MONGO_DB_NAME'))

  client = pymongo.MongoClient(mongo_connection)
  database = client['todoapp-DB']
```

Then your other functions can swap `database['todoapp-Collection']` for `get_database()['todoapp-Collection']`

Reply:
This was not working for me and due to time restrictions, I have left this part of the code as it is.